### ISSUE TITLE
Apk repository

### ISSUE THREAD

**Issue by eliekh05** [ish-app/ish#2350]
my apk repo is apk.ish.app not alpine default repo can you fix it

latest version of ios ish

**Comment by saagarjha** [comment]
This is behaving as intended.

**Comment by eliekh05** [comment]
> This is behaving as intended.

but i know that the alpine official repo: http://dl-cdn.alpinelinux.org/alpine/v3.19/community

and

http://dl-cdn.alpinelinux.org/alpine/v3.19/main

**Comment by kuflierl** [comment]
> > This is behaving as intended.
> 
> but i know that the alpine official repo: http://dl-cdn.alpinelinux.org/alpine/v3.19/community
> 
> and
> 
> http://dl-cdn.alpinelinux.org/alpine/v3.19/main
> 
> and i posted same issue before on twitter/x or github i dont remember but no one has fixed it since and redoing it
> 
> and to have the all expected packages you need to change it or use ubuntu based with apt repo or something 

The repository apk.ish.app forks most of the apps in the alpine repository to patch them for compatibility. You are free to manually add the main repository but I would advise caution since they weren't built with ish in mind.

Which application are you interested in that isn't included in the ish repo? Maybe someone might go ahead and add it.

**Comment by eliekh05** [comment]
> > > This is behaving as intended.
> > 
> > but i know that the alpine official repo: http://dl-cdn.alpinelinux.org/alpine/v3.19/community
> > 
> > and
> > 
> > http://dl-cdn.alpinelinux.org/alpine/v3.19/main
> > 
> > and i posted same issue before on twitter/x or github i dont remember but no one has fixed it since and redoing it
> > 
> > and to have the all expected packages you need to change it or use ubuntu based with apt repo or something 
> 
> The repository apk.ish.app forks most of the apps in the alpine repository to patch them for compatibility. You are free to manually add the main repository but I would advise caution since they weren't built with ish in mind.
> 
> Which application are you interested in that isn't included in the ish repo? Maybe someone might go ahead and add it.

lxqt desktop and webmin and xrdp and other gui/servers or all alpine packages

**Comment by kuflierl** [comment]
> > > > > > This is behaving as intended.
> > > > > 
> > > > > 
> > > > > but i know that the alpine official repo: http://dl-cdn.alpinelinux.org/alpine/v3.19/community
> > > > > and
> > > > > http://dl-cdn.alpinelinux.org/alpine/v3.19/main
> > > > > and i posted same issue before on twitter/x or github i dont remember but no one has fixed it since and redoing it
> > > > > and to have the all expected packages you need to change it or use ubuntu based with apt repo or something
> > > > 
> > > > 
> > > > The repository apk.ish.app forks most of the apps in the alpine repository to patch them for compatibility. You are free to manually add the main repository but I would advise caution since they weren't built with ish in mind.
> > > > Which application are you interested in that isn't included in the ish repo? Maybe someone might go ahead and add it.
> > > 
> > > 
> > > lxqt desktop and webmin and xrdp and other gui/servers or all alpine packages
> > 
> > 
> > I am not quite sure what you mean. xrdp and lxqt-desktop were both in the repository. Webmin doesn't even have a repository on alpine https://pkgs.alpinelinux.org/packages?name=webmin&branch=v3.19&repo=&arch=&maintainer=. Also I made a mistake: ish apparently doesn't patch a thing. It simply holds back packages for a day or so.
> 
> then you don't know anything about ish you are just telling like you know but you are lying

Hey sorry mate, its just something i had heard but then after i wrote my comment i started having my suspicions. I then Investigated in the deployment script and found out. 
Update: i just realized that @saagarjha had already corrected my mistake before i did, thanks.

**Comment by kuflierl** [comment]
> > > > > > > > This is behaving as intended.
> > 
> > > > > > > 
> > 
> > > > > > > 
> > 
> > > > > > > but i know that the alpine official repo: http://dl-cdn.alpinelinux.org/alpine/v3.19/community
> > 
> > > > > > > and
> > 
> > > > > > > http://dl-cdn.alpinelinux.org/alpine/v3.19/main
> > 
> > > > > > > and i posted same issue before on twitter/x or github i dont remember but no one has fixed it since and redoing it
> > 
> > > > > > > and to have the all expected packages you need to change it or use ubuntu based with apt repo or something
> > 
> > > > > > 
> > 
> > > > > > 
> > 
> > > > > > The repository apk.ish.app forks most of the apps in the alpine repository to patch them for compatibility. You are free to manually add the main repository but I would advise caution since they weren't built with ish in mind.
> > 
> > > > > > Which application are you interested in that isn't included in the ish repo? Maybe someone might go ahead and add it.
> > 
> > > > > 
> > 
> > > > > 
> > 
> > > > > lxqt desktop and webmin and xrdp and other gui/servers or all alpine packages
> > 
> > > > 
> > 
> > > > 
> > 
> > > > I am not quite sure what you mean. xrdp and lxqt-desktop were both in the repository. Webmin doesn't even have a repository on alpine https://pkgs.alpinelinux.org/packages?name=webmin&branch=v3.19&repo=&arch=&maintainer=. Also I made a mistake: ish apparently doesn't patch a thing. It simply holds back packages for a day or so.
> > 
> > > 
> > 
> > > then you don't know anything about ish you are just telling like you know but you are lying
> > 
> > 
> > 
> > Hey sorry mate, its just something i had heard but then after i wrote my comment i started having my suspicions. I then Investigated in the deployment script and found out. 
> > 
> > Anyways it doesn't change the fact that the packages you said were missing from the repo aren't missing at all. Could you elaborate why you thought that?
> > 
> > Update: i just realized that @saagarjha had already corrected my mistake before i did, thanks.
> 
> @kuflierl my initial problem is that ish is using apk.ish.app but alpine official is http://dl-cdn.alpinelinux.org/alpine/v3.19/community and http://dl-cdn.alpinelinux.org/alpine/v3.19/main

Does it make a difference if the contents is the same?

**Comment by eliekh05** [comment]
> > > > > > > > > This is behaving as intended.
> > > 
> > > 
> > > > > > > >
> > > 
> > > 
> > > > > > > >
> > > 
> > > 
> > > > > > > > but i know that the alpine official repo: http://dl-cdn.alpinelinux.org/alpine/v3.19/community
> > > 
> > > 
> > > > > > > > and
> > > 
> > > 
> > > > > > > > http://dl-cdn.alpinelinux.org/alpine/v3.19/main
> > > 
> > > 
> > > > > > > > and i posted same issue before on twitter/x or github i dont remember but no one has fixed it since and redoing it
> > > 
> > > 
> > > > > > > > and to have the all expected packages you need to change it or use ubuntu based with apt repo or something
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > >
> > > 
> > > 
> > > > > > > The repository apk.ish.app forks most of the apps in the alpine repository to patch them for compatibility. You are free to manually add the main repository but I would advise caution since they weren't built with ish in mind.
> > > 
> > > 
> > > > > > > Which application are you interested in that isn't included in the ish repo? Maybe someone might go ahead and add it.
> > > 
> > > 
> > > > > >
> > > 
> > > 
> > > > > >
> > > 
> > > 
> > > > > > lxqt desktop and webmin and xrdp and other gui/servers or all alpine packages
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > >
> > > 
> > > 
> > > > > I am not quite sure what you mean. xrdp and lxqt-desktop were both in the repository. Webmin doesn't even have a repository on alpine https://pkgs.alpinelinux.org/packages?name=webmin&branch=v3.19&repo=&arch=&maintainer=. Also I made a mistake: ish apparently doesn't patch a thing. It simply holds back packages for a day or so.
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > > then you don't know anything about ish you are just telling like you know but you are lying
> > > 
> > > 
> > > Hey sorry mate, its just something i had heard but then after i wrote my comment i started having my suspicions. I then Investigated in the deployment script and found out.
> > > Anyways it doesn't change the fact that the packages you said were missing from the repo aren't missing at all. Could you elaborate why you thought that?
> > > Update: i just realized that @saagarjha had already corrected my mistake before i did, thanks.
> > 
> > 
> > @kuflierl my initial problem is that ish is using apk.ish.app but alpine official is http://dl-cdn.alpinelinux.org/alpine/v3.19/community and http://dl-cdn.alpinelinux.org/alpine/v3.19/main
> 
> Does it make a difference if the contents is the same?

yes because first thing when go to apk.ish.app it will say welcome to nginx this means that the domain is not working so i need a working domain so thats why i opened the issue in the first place

**Comment by kuflierl** [comment]
That is a valid issue, it seems the developers didn't take their time replacing the main index file. But the repository does work.

**Comment by eliekh05** [comment]
> That is a valid issue, it seems the developers didn't take their time replacing the main index file. But the repository does work.

is there any way to pin the official alpine repositories http://dl-cdn.alpinelinux.org/alpine/v3.19/community and http://dl-cdn.alpinelinux.org/alpine/v3.19/main instead of apk.ish.app

**Comment by kuflierl** [comment]
i mean i gain absolutely noting if you do so but its pretty simple. You replace the file contents of `/etc/apk/repositories` with
```
http://dl-cdn.alpinelinux.org/alpine/v3.19/main
http://dl-cdn.alpinelinux.org/alpine/v3.19/community
```
and running `rm -rf /ish`
